services/issue/comments.go

```diff
@@ -53,7 +53,7 @@ func CreateRefComment(doer *user_model.User, repo *repo_model.Repository, issue
 		return nil
 	}
 
-	_, err = CreateComment(&issues_model.CreateCommentOptions{
+	_, err = CreateComment(db.DefaultContext, &issues_model.CreateCommentOptions{
```

Comment:
Hmm… Or we simply give this function a `ctx` as well…

Reply:
I have tried but it causes too many changes.

So just make it like [the line above](https://github.com/go-gitea/gitea/pull/24068/files/d528337bc366afb53a36e8736605e098fc2ed6b3#diff-cacc2474f9155102033f0df86a83e1c5019830bed4118c352f8bd6ff8c441c8bL45) (also use db.DefaultContext)

Leave it to next round refactoring.